decidim-admin/app/views/decidim/admin/organization_external_domain_whitelist/edit.html.erb

```diff
@@ -1,13 +1,13 @@
 <% add_decidim_page_title(t("edit_external_domains", scope: "decidim.admin.titles")) %>
 
 <div class="item_show__header">
-  <h2 class="item_show__header-title">
+  <h1 class="item_show__header-title">
     <%= t("title", scope: "decidim.admin.organization_external_domain_whitelist.form") %>
-  </h2>
+  </h1>
 </div>
 
 <div class="item__edit-form">
-  <%= decidim_form_for(@form, html: { class: "form-defaults form edit_organization_external_domain_whitelist" }, url: organization_external_domain_whitelist_path, method: :patch) do |f| %>
+  <%= decidim_form_for(@form, html: { class: "form-defaults form edit_organization_external_domain_allowlist" }, url: organization_external_domain_allowlist_path, method: :patch) do |f| %>
```

Comment:
#12287 wasn't backported so this change shouldn't be here 
```suggestion
  <%= decidim_form_for(@form, html: { class: "form-defaults form edit_organization_external_domain_whitelist" }, url: organization_external_domain_whitelist_path, method: :patch) do |f| %>
```

Reply:
Fixed by [1157d28](https://github.com/decidim/decidim/pull/12429/commits/1157d28e66d88e5acbbd8d501767bb249b24648c)